gatsby-node.js

```diff
@@ -189,7 +189,7 @@ exports.onCreateNode = ({node, boundActionCreators, getNode}) => {
           const day = match[3];
           const filename = match[4];
 
-          slug = `/blog/${year}/${month}/${day}/${filename}.html`;
+          slug = `blog/${year}/${month}/${day}/${filename}.html`;
```

Comment:
This change (removing the "/" prefix) breaks the main Docs redirect.

To reproduce, click on "Blog" in the top nav, which should redirect to the most recent blog entry, but now/instead redirects to "/blog/blog/2017/09/26/react-v16.0.html" (note the double "blog/blog" at the beginning).